packages/frontend/src/document/model_document_editor.css

```diff
@@ -20,3 +20,44 @@
 .model-theory select:invalid {
     color: darkgray;
 }
+
+.selector {
+    background-color: rgb(245, 245, 245);
+    color: rgb(73, 73, 73);
+    border: none;
+    font-size: var(--h3-font-size);
+    font-family: inter;
+    padding: 14px;
+    border-radius: 3%;
+}
+
+.description {
+    color: darkgray;
+    font-size: 11px;
+    text-align: left;
+    line-height: 1px;
+}
+
+#divisionCategoryHeader {
```

Comment:
We're using "hypthen-case" in CSS files, so this should be "division-category-header."